.github/workflows/generate-manifest.yml

```
@@ -0,0 +1,81 @@
+          commit-message: |
+            feat: add manifest for ${{ steps.repo-info.outputs.repo_name }}
+            
+            Generated manifest JSON for repository: ${{ github.event.inputs.repo_url }}
+            
+            🤖 Generated with [Claude Code](https://claude.ai/code)
+            
+            Co-Authored-By: Claude <noreply@anthropic.com>
+          title: "feat: Add MCP manifest for ${{ steps.repo-info.outputs.repo_name }}"
+          body: |
+            ## Summary
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)